crates/xilem_masonry/examples/flex.rs

```diff
@@ -0,0 +1,42 @@
+// Copyright 2024 the Xilem Authors
+// SPDX-License-Identifier: Apache-2.0
+
+use masonry::widget::{CrossAxisAlignment, MainAxisAlignment};
+use winit::error::EventLoopError;
+use xilem::{
+    view::{button, flex, label},
+    MasonryView, Xilem,
+};
+
+struct AppState {
+    count: u32,
+}
+
+impl AppState {
+    fn decrement(&mut self) {
```

Comment:
Looking at this code again - I'm not sure I like these two methods; I think these methods would probably just be closures inline in the `app_logic`.

It's nice to keep the examples light on ceremony

Reply:
Good call, this started out as my own demo that I copied in. Should I just nuke the counter logic altogether since this is just showing the layout?

Reply:
This has been slimmed down considerably now